packages/ragbits-core/tests/unit/utils/pyproject/testprojects/bad_factory_project/pyproject.toml

```diff
@@ -2,6 +2,6 @@
 name = "bad_factory_project"
 
 [tool.ragbits.core.default_llm_factories]
-non_existing = "ragbits.core.llms.factory.simple_litellm_factory"
-vision = "ragbits.core.llms.factory.simple_litellm_vision_factory"
-structured_output = "ragbits.core.llms.factory.simple_litellm_vision_factory"
\ No newline at end of file
+non_existing = "ragbits.core.llms.factory:simple_litellm_factory"
+vision = "ragbits.core.llms.factory:simple_litellm_vision_factory"
+structured_output = "ragbits.core.llms.factory:simple_litellm_vision_factory"
```

Comment:
new line

Reply:
This PR didn't change anything in that regard. I just mechanically changed the format for giving LLM factories in the entire project. The end of the file was left exactly as it was before.

Reply:
ok, this is not related to the change, but we can add this new line at the end of the file to remove this annoying gh warning

Reply:
By "annoying gh warning" you mean the red circle that symbolise end of file without new line? Ok, I'll do it, but if that's something we are serious about, we should just configure a linter instead

Reply:
Done

Reply:
totally agree with the linter part